docs/source/sycamore/tutorials/etl_for_opensearch.md

```diff
@@ -1,10 +1,12 @@
-# Using Sycamore's data prep libraries locally
+# Process and load data into an OpenSearch hybrid search index
 
-This tutorial provides a walkthrough of how to use Sycamore's data preparation libraries locally to prepare, enhance, and embed a PDF dataset from S3 and load it into a local Sycamore stack. We encourage to go over [Sycamore's architecture](../welcome_to_sycamore/architecture.md) before you work through this tutorial. We will be using unstructured PDFs from the [Sort Benchmark](http://sortbenchmark.org/) website. This data is publicly available in S3 here, so you do not need AWS credentials to access it: `s3://aryn-public/sort-benchmark/pdf/`
+This tutorial provides a walkthrough of how to use Sycamore to extract, enrich, transform, and create vector embeddings from a PDF dataset in S3 and load it into OpenSearch. The way in which you run ETL on these document is critical for the end quality of your application, and you can easily use Sycamore to facilitate this. The example below shows a few transforms Sycamore can do.
+
+We will be using PDF documents from the [Sort Benchmark](http://sortbenchmark.org/) website. This data is publicly available in S3 here, so you do not need AWS credentials to access it: `s3://aryn-public/sort-benchmark/pdf/`
```

Comment:
I think this is last line is a bit confusing, but I might be missing something. If they aren't using aws creds to access the data I think we'll need to give them the entire file path.

If I go to http://aryn-public.s3.amazonaws.com/sort-benchmark/pdf I don't get anything back but if I go to:
https://aryn-public.s3.amazonaws.com/sort-benchmark/pdf/2004_Nsort_Minutesort.pdf I can see the file. I think if they need to walk the entire directory and get all the files then they'll need s3 creds.